cfn-resources/project/cmd/resource/resource.go

```diff
@@ -181,7 +184,7 @@ func Update(req handler.Request, prevModel *Model, currentModel *Model) (event h
 		return event, nil
 	}
 
-	if currentModel.Name != nil {
+	if currentModel.Name != nil || !reflect.DeepEqual(prevModel.Tags, currentModel.Tags) {
```

Comment:
are we sure prevModel receives a value? We don't usually use it but might be good to use so I am curious

Reply:
related to that, contract testing might behave differently from Prod (e.g. passing different info in prevModel), so it's hard to know without publishing it

Reply:
Yes, I'm pretty sure.
Since the `model.go` has:
```golang
type Model struct {
  // other fields
  Tags                      map[string]string `json:",omitempty"`
}
```

Even if it would be `nil` and the other map empty `reflect.DeepEqual` would work:
```golang
func TestDeepEqualNil(t *testing.T) {
	assert.True(t, reflect.DeepEqual(nil, nil))
	assert.False(t, reflect.DeepEqual(nil, map[string]string{}))
}
```

Reply:
@maastha ok to resolve?